rma_delivery/models/rma.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo import models
+
+
+class Rma(models.Model):
+    _inherit = "rma"
+
+    def _prepare_returning_picking(self, picking_form, origin=None):
+        """Gather the company option for default carrier on RMA returns. We could
+        either
+        """
+        super()._prepare_returning_picking(picking_form, origin)
+        strategy = picking_form.company_id.rma_delivery_strategy
+        delivery_method = picking_form.company_id.rma_fixed_delivery_method
+        partner_method = (
+            picking_form.partner_id.property_delivery_carrier_id
+            or picking_form.partner_id.commercial_partner_id.property_delivery_carrier_id
+        )
+        if (
+            strategy == "customer_method"
+            or strategy == "mixed_method"
+            and partner_method
```

Comment:
Make sure of the operator precedence

```suggestion
            or (strategy == "mixed_method" and partner_method)
```

Reply:
done